### DISCUSSION TITLE
What happens when my destination drive runs out of space?

### DISCUSSION THREAD

**Discussion by manthanr** [swar/Swar-Chia-Plot-Manager#163] [Q&A]
1. Does Plot Manager detect that the HDD is out of space and pauses the plotting, or will it still attempt to move completed plots to the destination drive and error out? 
2. Also, is it possible to modify the configuration and restart Plot Manager to change the destination directory (without interrupting plots in progress)?

**Comment by actualLocke** [comment]
I'm very sad to see no one answered your question. I'll let you know what happens......

**Reply by Cminer3207** [reply]
Simple. 
Just set Maximum plots Value to stop just before your Destination Directiry is full. 
A plot is 101.3 GiB (about 109GB)... Divide Drive size by 109... RoundDown... 

Then set a timer based on your average plot time so you are watching when it finishes. 

Change Destination Drive and restart.

**Reply by actualLocke** [reply]
Well good news and bad news. The good news is I was able to divert disaster. The bad news is we still don't know what will happen.

And I disagree. SWAR is awesome. A few things that could be better sure.
